src/imgui/ImGuiReverseBar.cc

```diff
@@ -190,22 +190,52 @@ void ImGuiReverseBar::showMenu(MSXMotherBoard* motherBoard)
 				ImGui::TextUnformatted("No replays found"sv);
 			} else {
 				ImGui::TextUnformatted("Select replay"sv);
-				im::ListBox("##select-replay", [&]{
-					for (const auto& [fullName_, displayName_, _] : replayNames) {
+				int flags = ImGuiTableFlags_RowBg |
+					ImGuiTableFlags_BordersV |
+					ImGuiTableFlags_BordersOuter |
+					ImGuiTableFlags_Resizable |
+					ImGuiTableFlags_Sortable |
+					ImGuiTableFlags_Hideable |
+					ImGuiTableFlags_Reorderable |
+					ImGuiTableFlags_ContextMenuInBody |
+					ImGuiTableFlags_SizingStretchProp;
+				im::Table("##select-replay", 2, flags, {}, [&]{
+					for (const auto& [fullName_, displayName_, ftime_] : replayNames) {
 						const auto& fullName = fullName_; // clang workaround
-						const auto& displayName = displayName_; // clang workaround
-						if (ImGui::Selectable(displayName.c_str())) {
-							manager.executeDelayed(makeTclList("reverse", "loadreplay", fullName));
-						}
-						im::PopupContextItem([&]{
+						if (ImGui::TableNextColumn()) {
+							const auto& displayName = displayName_; // clang workaround
+							if (ImGui::Selectable(displayName.c_str(), false, ImGuiSelectableFlags_SpanAllColumns | ImGuiSelectableFlags_AllowOverlap)) {
+								manager.executeDelayed(makeTclList("reverse", "loadreplay", fullName));
+							}						im::PopupContextItem([&]{
 							if (ImGui::MenuItem("delete")) {
 								confirmCmd = makeTclList("file", "delete", fullName);
 								confirmText = strCat("Delete replay '", displayName, "'?");
 								openConfirmPopup = true;
 							}
 						});
-					}
-				});
+
+						}
+						if (ImGui::TableNextColumn()) {
+							const auto& ftime = ftime_; // clang workaround
+
+							// Convert file_time_type to system_clock::time_point (the standard clock since epoch)
+							auto sctp = std::chrono::time_point_cast<std::chrono::system_clock::duration>(
+									ftime - std::filesystem::file_time_type::clock::now() +
+									std::chrono::system_clock::now());
+
+							// Convert system_clock::time_point to time_t (time since Unix epoch in seconds)
+							std::time_t cftime = std::chrono::system_clock::to_time_t(sctp);
+
+							// Convert time_t to local time (broken-down time in the local time zone)
+							std::tm* local_time = std::localtime(&cftime);
+
+							// Get the local time in human-readable format
+							std::stringstream ss;
+							ss << std::put_time(local_time, "%F %T");
+							ImGui::TextUnformatted(ss.str().c_str());
```

Comment:
I haven't tried your patch yet, so I haven't seen how it actually looks. But I think it might be a good idea to show only one of the date/time (depending on whether date==today). Many other programs (e.g. file managers) do this as well. Reason is to keep the table more compact (it's a sub menu after all, and those shouldn't be too big).

Reply:
The biggest problem is that the whole list is way too long (as I put in the commit message). Well, two issues:
1. the first time it seems to appear *behind* the openMSX window: 
![image](https://github.com/user-attachments/assets/940fbd1b-06ae-4949-b0cd-d10289029291)
2. and the 2nd time on the place you expect. But yeah, way too long:
![image](https://github.com/user-attachments/assets/32305c9a-eec5-4f76-a28f-7116501cde52)

The first problem seems to be some more generic Z-level issue, more GitHub issues have mentioned it. No idea what to do there.
The fact that it is so long, what do you recommend to do about it? The size of the original listbox would be fine.

As for the date/time, I think it looks fine, even without refining this. What do you think?

Reply:
For the ListBox we passed the size: `ImVec2(ImGui::GetFontSize() * 20.0f, 240.0f)`.
We could do the same for the table. See `src/imgui/ImGuiCpp.hh` lines 454-474. There are various overloads of `im::Table()`, the size is the 4th (optional) parameter.

Reply:
OK, I tried that, but it did not have any effect at all. What am I doing wrong? I simply passed this exact same size as the 4th parameter, but I still get the super long table... Yes, I'm sure I compiled it etc.

Reply:
Ah, the width does has effect. But the height does not.

Reply:
OK, setting ScrollY flag helps. It behaves a bit weird when you get to the end of the list though... Try to scroll in the last item... I can't really do it somehow.

Reply:
Could it be you're testing with a non-assert build? Because when I try your PR, I immediately trigger an assert (with a semi-helpful message explaining what's wrong).
You need to move the line containing `TableSetupScrollFreeze()` before the lines with `TableSetupColumn()` (see also various examples in our code, or in `imgui_demo.cc`).

Reply:
Thanks, solved and from now on using devel builds :)